spec/features/user_can_signup_spec.rb

```diff
@@ -1,80 +1,107 @@
 require 'rails_helper'
 
-RSpec.feature "Signup", type: :feature do
-  context "Form data" do
+RSpec.feature 'Signup', type: :feature do
+  context 'Form data' do
 
-    scenario "it has an email field" do
+    scenario 'it has an email field' do
       go_to_homepage
 
       expect(page).to have_field("user[email]") 
     end
 
-    scenario "it has a password field" do
+    scenario 'it has a password field' do
       go_to_homepage
 
       expect(page).to have_field("user[password]") 
     end
+
+    scenario 'it has a name field' do
+      go_to_homepage
+
+      expect(page).to have_field("user[name]") 
+    end
+
+    scenario 'it has a password confirmation field' do
+      go_to_homepage
+
+      expect(page).to have_field('user[password_confirmation]') 
+    end
   end
 
-  context "User must fill in a valid password" do
-    scenario "user left password field empty" do
-      sign_up(email: "test@test.com", password: "")
+  context 'User must fill in a valid password' do
+    scenario 'user left password field empty' do
+      sign_up(name: 'test', email: "test@test.com", password: '', password_confirmation: '')
 
-      expect(page).to have_content("Password can't be blank")
-      expect(page).to have_content("Password is too short (minimum is 6 characters)")
+      expect(page).to have_content('Password can\'t be blank')
+      expect(page).to have_content('Password is too short (minimum is 6 characters)')
     end
 
     scenario "user provides a password sorter than 6 characters" do
-      sign_up(email: "test@test.com", password: "12345")
```

Comment:
typo heaven! check the word "sorter" in the scenario

Reply:
Not ours! :)